coalib/output/Logging.py

```diff
@@ -4,25 +4,48 @@
 import logging
 import logging.config
 
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
 
-def configure_logging():
+
+def configure_logging(new_log_level=logging.INFO, incremental=False):
     """
     Configures the logging with hard coded dictionary.
+
+    If the incremental param is ``True``, all the log-handlers configuration
+    other than level will be ignored (including class).
+    Thus, use it only when changing the log_level.
+    When changing log_level, ``json-handler``, ``console-handler``,
+    and ``file-handler`` should all exist.
+    ``json-handler`` is created as a null handler to avoid this error.
+
+    :param new_log_level:
+        The new log level to set for json and console handlers.
+    :param incremental:
+        ``False`` if setting up new loggers, ``True`` if only changing
+        the log level.
+    :return:
+        The new_log_level is returned.
```

Comment:
We need a variable in the log_printer module which can tell us the level currently set, that is why I am returning the log level and saving it in a variable called `self.level` in the log_printer. https://github.com/coala/coala/pull/3740/files#diff-f9ad1a606d13482370eb9a8e9f3065aeR143